src/perception/lidar_detection/lidar_detection/complex_yolov4/README.md

```diff
@@ -0,0 +1,11 @@
+# Complex YOLOv4
+
+The PyTorch Implementation based on YOLOv4 of the paper: [Complex-YOLO: Real-time 3D Object Detection on Point Clouds](https://arxiv.org/pdf/1803.06199.pdf)
+
+Most files are taken (some modified) from the GitHub Repo: [https://github.com/maudzung/Complex-YOLOv4-Pytorch/tree/master](https://github.com/maudzung/Complex-YOLOv4-Pytorch/tree/master)
+
+## Installation and Setup
+
+Follow the instrucitons below to ensure the model is setup properly:
+1. Click [here](https://drive.google.com/drive/folders/16zuyjh0c7iiWRSNKQY7CnzXotecYN5vc) and download the `complex_yolov4_mse_loss.pth` file
+2. Move `complex_yolov4_mse_loss.pth` into the directory `navigator/data/perception/checkpoints/complex_yolov4_mse_loss.pth`
```

Comment:
Is it possible to change the path to navigator/data/perception/models? It's where we've historically kept our model weight files

Reply:
I'll move both Ragib's and my files to that directory then